package.json

```diff
@@ -42,7 +44,7 @@
     "expo": "^33.0.0",
     "moment": "^2.24.0",
     "react": "16.8.3",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-33.0.0.tar.gz",
+    "react-native": "0.60.0",
```

Comment:
That's not going to work. This is an Expo project and requires Expo's fork of React Native.